### PR TITLE
Implementing the Input Interface for inputing a list of websites

### PR DESCRIPTION
**Pull request by Blindspot22**

### CODE REVIEW

**Comment by Christiantyemele** [comment]
please sir you close our ticket without us making a pull request

**Comment by stephane-segning** [comment]
> please sir you close our ticket without us making a pull request 

Where is your pull request @Christiantyemele? Please past the link of it after this comment.

**Comment by Christiantyemele** [comment]
the issues is that you closed and merge our ticket without us(Scott and i)  even making a pull request

**Comment by Blindspot22** [comment]
We will then do another pull request

**Comment by Christiantyemele** [comment]
@stephane-segning resolved merge conflicts

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Please remove the `.idea` folder and exclude it using `.gitignore`

**Comment by Christiantyemele** [comment]
@stephane-segning  .idea folder placed in the .gitignore file

**Comment by Christiantyemele** [comment]
@stephane-segning  modified the input.html file to meet the new specificities

**Comment by Christiantyemele** [comment]
@stephane-segning requesting a review to approve merging

File: docs/input-field.md
Comment (stephane-segning) [inline docs/input-field.md:1]: Why did you deleted this file? Please cancel this change

**Review by stephane-segning — CHANGES_REQUESTED** [review]
Please check this

**Comment by Christiantyemele** [comment]
@stephane-segning sir  i did not remove docs/input-field.md

File: templates/input.html
Comment (Awambeng) [inline templates/input.html:1]: Hello guys, you did a good job but i think there are some slight changes to review. When i tested your interface i noticed the following:
- The input and depth of scrapping field text are white making the user not seeing what he or she types
- Finally, the user should scrape both images and number of text not asking he or she to choose an option

**Review by Awambeng — CHANGES_REQUESTED** [review]

File: templates/input.html
Comment (Blindspot22) [reply templates/input.html:1]: Ok just give me a sec

Comment (Blindspot22) [reply templates/input.html:1]: @stephane-segning and @Awambeng  Please review the new modifications

**Review by Awambeng — APPROVED** [review]
Code verified and confirmed @stephane-segning you have the final say.

**Comment by stephane-segning** [comment]
Thank you @Awambeng. But please @Blindspot22, remove changes that are not yours

**Comment by Blindspot22** [comment]
Thank you @stephane-segning and @Awambeng Hope all is good apart from the changes that have to be removed ???

**Comment by stephane-segning** [comment]
@Blindspot22 I can see unrelated changes here:
Screenshot 2023-11-30 at 12 04 02

and here:
Screenshot 2023-11-30 at 12 04 48

and here too
Screenshot 2023-11-30 at 12 05 04

Ask @Awambeng how to cancel changes that are not related

**Comment by Christiantyemele** [comment]
@stephane-segning  i confirm the code in the PR is same as the local repository

**Review by stephane-segning — APPROVED** [review]
Good work @Awambeng @Blindspot22 @Christiantyemele ! This was a difficult one :)

**Pull request merged**